app/helpers/users_helper.rb

```diff
@@ -1,3 +1,6 @@
 # frozen_string_literal: true
 module UsersHelper
+  def get_first_name(user)
```

Comment:
remove this method
add "first_name" and "last_name" columns to User model
and take them from social networks auth.first_name (something like this) during registration